src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/RetailPriceType.php

```diff
@@ -166,15 +187,26 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'class' => 'retail-price-tax-rules-group-id',
                 ],
                 'label' => $this->trans('Tax rule', 'Admin.Catalog.Feature'),
-                'help' => !$this->isTaxEnabled ? $this->trans('Tax feature is disabled, it will not affect price tax included.', 'Admin.Catalog.Feature') : '',
+                'help' => !$this->isTaxEnabled ?
+                    $this->trans('Tax feature is disabled, it will not affect price tax included.', 'Admin.Catalog.Feature')
+                    : $this->trans(
+                        $taxGroupHelpText,
+                        'Admin.Catalog.Feature',
+                        ['%1$s' => Country::getIsoById($this->contextCountryId), '%2$s' => $selectedTaxRate, '%3s$s' => $stateIsoCode]
+                    ),
+                'help_attr' => ['class' => 'js-tax-rule-help',
```

Comment:
Also, I'm quite surprised that you can add a `help_attr` so easily without handling it in a form extension because the `help` option was added by us here  https://github.com/PrestaShop/PrestaShop/blob/ff96226eae149d58b34ca6c5228b846595d05914/src/PrestaShopBundle/Form/Admin/Extension/HelpTextExtension.php#L47

Does it mean Symfony already handles a native `help_attr` option? 🤔

Reply:
Oh, actually it does!! https://github.com/symfony/form/blob/4.4/Extension/Core/Type/FormType.php#L198

It also handles the `help` option which means our extension is probably completely overkilled 😅